tests/test_ops/test_onnx.py

```diff
@@ -345,32 +345,6 @@ def warpped_function(torch_input, torch_rois):
         assert np.allclose(pytorch_output, onnx_output, atol=1e-3)
 
 
```

Comment:
update the onnx [doc](https://github.com/grimoire/mmcv/blob/ops_document/docs/onnx.md) as well.